grid/app/main/events/dfl/control_events.py

```diff
@@ -0,0 +1,71 @@
+import json
+from flask_login import login_user
```

Comment:
The grouping "standard python, external module, local" is being partly acknowledge here .